plutus-core/plutus-ir/test/transform/inline/nameCapture.golden

```diff
@@ -0,0 +1,10 @@
+(let
+  (nonrec)
+  (termbind (strict) (vardecl a (con integer)) (con integer 10))
+  (termbind (strict) (vardecl c (con integer)) [ [ (builtin addInteger) a ] a ])
+  (let
+    (nonrec)
+    (termbind (strict) (vardecl a (con integer)) (con integer 7))
+    [ [ (builtin addInteger) [ [ (builtin addInteger) a ] a ] ] a ]
```

Comment:
Oh noes!!!

Reply:
The latest 2 commits show that the uniques are different.

Reply:
`rename` should give fresh [names](https://github.com/input-output-hk/plutus/blob/760d54810e05e02c7c538d5e82b7a86a0fc9b3ec/plutus-core/plutus-core/src/PlutusCore/Rename.hs#L38) too it seems? I'm looking into it.